### DISCUSSION TITLE
How to access to GTD views from anywhere

### DISCUSSION THREAD

**Discussion by vanhtuan0409** [nvim-neorg/neorg#223] [Q&A] [answered]
Greeting,

May I know how I can access to GTD task list when I'm working? Currently, I'm only able to call `Neorg` when a `norg` file is opened. How can I open the GTD when I'm in a source code project?

**Comment by danymat** [comment, marked as answer]
Hello, GTD rely on Neorg to operate. 
So you need to start Neorg before doing any GTD actions.

You can start Neorg in silent mode (without opening a norg file) by doing: `NeorgStart silent=true`
Afterwards, you can use GTD commands freely !
